core/src/main/java/nl/nn/adapterframework/jdbc/migration/Migrator.java

```diff
@@ -74,15 +73,15 @@ public synchronized void configure(String configurationName, ClassLoader classLo
 				JdbcConnection connection = new JdbcConnection(getConnection());
 				instance = new LiquibaseImpl(ibisContext, cl, connection, configurationName, changeLogFile);
 			}
-			catch (JdbcException e) {
-				throw new ConfigurationException("migrator error connecting to database ["+dataSource+"]", e);
-			}
 			catch (ValidationFailedException e) {
 				throw new ConfigurationException("liquibase validation failed", e);
 			}
 			catch (LiquibaseException e) {
 				throw new ConfigurationException("liquibase failed to initialize", e);
 			}
+			catch (Throwable e) {
```

Comment:
Van waar deze bazooka?

Reply:
Ergens diep onderwater gooit Liquibase een RuntimeException. Ik dacht voor de zekerheid als het niet lukt, vang ik deze excepties, anders laad de configuratie niet in.